api/api.yaml

```diff
@@ -39,6 +39,87 @@ paths:
         '500':
           $ref: '../result/request.yaml#/components/responses/InternalServerError'
           x-field-uid: 3
+    patch:
+      tags: ['Configuration']
+      operationId: update_config
+      description: >-
```

Comment:
We have plan to incorporate `update_config` within this PR

Reply:
If `update_config` is already supported for a given implementation, then yes, we should incorporate it.